python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -827,7 +829,14 @@ def runDataRef(self, dataRefList):
             result = self.extract.run(exposure)
             ratios.append(result.outputRatios)
 
-        finalResults = self.solver.run(ratios, camera=camera)
+        for detIter, detector in enumerate(camera):
+            if detector.getName() == activeChip:
+                detectorId = detIter
+        outputDims = {'instrument': camera.getName(),
+                      'detector': detectorId,
+                      }
+
+        finalResults = self.solver.run(ratios, camera=camera, outputDims=outputDims)
```

Comment:
Why were the output dimensions not needed before, but now they are being passed?  What difference does it make?

Reply:
I didn't think they were needed, because I had thought that the `calibChip` was the name, and so could be used to set the `_detectorName` attribute.  Constructing an `outputDims` for gen2 also reduces possible differences.